README.md

```diff
@@ -83,6 +83,13 @@ By selecting quick replies instead of typing manually, agents/users can respond
 -   **`/quick ai`**: Use AI to generate replies
 -   **`/quick help`**: Get help with Quick Reply
 -   **`/qs <reply name>`**: Quickly search and send a reply by name
+-   **`/quick create "<name>" <message>`**:  Create a quick reply directly from the input box with a name and message
+
+#### Example:
+```sh
+/quick create "greeting" Hello! How have you been?
```

Comment:
This is not what I ment 

How double quotes make the multiple words a single text for body use